test/unit/functionsAsyncAwait.test.js

```diff
@@ -0,0 +1,226 @@
+const {
+  getFilteredsStargazers,
+  getTopFive,
+  sortDateData,
+  getSum,
+  getSumOfReposStars,
+  getReposMoreThanFiveStars,
+  getLastUpdatedRepos,
+} = require("../../scripts/functionsAsyncAwait.js");
+
+const { data } = require("../fixtures/data.js");
+
+describe("Test of functions to get the repos with more than five stars", () => {
```

Comment:
It is kind of common to have the first describe as the function name that you are testing in this case:
```suggestion
describe("getFilteredsStargazers", () => {
```
Do this for all the tests please